serverless.ts

```diff
@@ -35,12 +37,8 @@ const serverlessConfiguration: AWS = {
       cors: true
     },
     environment: {
-      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
-      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
-      ELASTIC_URL: env.ELASTIC_URL,
-      ELASTIC_APP_ENGINE: env.ELASTIC_APP_ENGINE,
-      ELASTIC_ADMIN_USERNAME: env.ELASTIC_ADMIN_USERNAME,
-      ELASTIC_ADMIN_PASSWORD: env.ELASTIC_ADMIN_PASSWORD
```

Comment:
Are these applied from somewhere else?

Reply:
Yes, it looks like they are coming from the config/index.ts. Perhaps this was done so its using envalid.